command/crypto/jws/sign.go

```diff
@@ -201,7 +204,23 @@ func signAction(ctx *cli.Context) error {
 			return errs.MutuallyExclusiveFlags(ctx, "x5c-cert", "jwks")
 		}
 		isX5C = true
-	} else {
+	}
+
+	var isX5T bool
+	if len(x5tCertFile) > 0 {
+		if len(x5tKeyFile) == 0 {
```

Comment:
I assume that `x5c-cert` and `x5t-cert` are mutually exclusive. Let's add that invariant check to both "if" blocks (even though we only need it in one - but for posterity).